packages/frontend/core/src/hooks/use-journal.ts

```diff
@@ -162,6 +163,7 @@ export const useJournalRouteHelper = (docCollection: DocCollection) => {
       workbench.openDoc(page.id, {
         at: newTab ? 'new-tab' : 'active',
       });
+      track.$.navigationPanel.journal.journalVisited();
```

Comment:
Tracking here may not be precise, since it could be opened from page list, or through a link within a doc

Reply:
`navigate` is the event for page redirecting, avoid bring more event variant with same meaning